marlowe-dashboard-client/src/Play/State.purs

```diff
@@ -65,12 +71,27 @@ mkInitialState walletDetails currentSlot timezoneOffset =
   , contractsState: ContractHome.defaultState
   }
 
+handleQuery ::
+  forall m.
+  CombinedWSStreamToClient -> HalogenM MainFrame.State MainFrame.Action ChildSlots Msg m Unit
+handleQuery (InstanceUpdate contractInstanceId instanceStatusToClient) = pure unit
+
+handleQuery (SlotChange slot) = assign (_playState <<< _currentSlot) $ toFront slot
```

Comment:
We should unify `SlotChange` and `SetCurrentSlot` as they probably compete to set the current slot. When we do, it's important to keep in mind that `SetCurrentSlot` ends up calling `AdvanceTimedOutContracts` which is responsible for "creating" the timeout steps

Reply:
yes, this does seem to be causing problems